plugins/experimental/ssl_session_reuse/src/ats_ssl_plugin.cc

```diff
@@ -0,0 +1,60 @@
+/** @file
+
+  ats_ssl_plugin.cc - plugin setup
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+ */
+
+#include <stdio.h>
+#include <ts/ts.h>
+#include <ts/apidefs.h>
+#include <openssl/ssl.h>
+
+#include "ssl_utils.h"
+int SSL_session_callback(TSCont contp, TSEvent event, void *edata);
+void
+TSPluginInit(int argc, const char *argv[])
```

Comment:
you can also just write:

TSPlugin(int, const char *argv[])

to indicate the first parameter is not used (without the (void)argc; line).